### PR TITLE
New Hairstyles!

### PR DESCRIPTION
**Pull request by BeeRobynn**
## About the PR
Adds new Hairdo's for the crew to show off! 

## Why / Balance
The crew has been having a bad hair day, so lets give them some more variety

## Technical details
Adds 21 new hairstyles ingame
Sprites lovingly handmade by bloomyex

## Media

image
image
image

## Breaking changes
N/A

**Changelog**
add: Added 21 new hairstyles!

### CODE REVIEW

File: Resources/Locale/en-US/accessories/human-hair.ftl
Comment (Piras314) [inline Resources/Locale/en-US/accessories/human-hair.ftl:1]: These changes need to be moved to Resources/Locale/en-US/Goobstation/accessories/human-hair.ftl, which you created, but is empty...

File: Resources/Prototypes/Entities/Mobs/Customization/Markings/human_hair.yml
Comment (Piras314) [inline Resources/Prototypes/Entities/Mobs/Customization/Markings/human_hair.yml:1]: Move these into Resources/Prototypes/Goobstation/Entities/Mobs/Customization/Markings/human_hair.yml please

**Review by Piras314 — CHANGES_REQUESTED** [review]
Some things need to be moved, otherwise this looks good.

**Review by Piras314 — APPROVED** [review]
looks good now 👍

File: Resources/Prototypes/Entities/Mobs/Customization/Markings/human_hair.yml
Comment (Aidenkrz) [reply Resources/Prototypes/Entities/Mobs/Customization/Markings/human_hair.yml:1]: ^^ move this shit

Comment (Piras314) [reply Resources/Prototypes/Entities/Mobs/Customization/Markings/human_hair.yml:1]: @Aidenkrz it was moved already

Comment (Aidenkrz) [reply Resources/Prototypes/Entities/Mobs/Customization/Markings/human_hair.yml:1]: so it is

**Pull request merged**